tox.ini

```diff
@@ -63,6 +63,18 @@ setenv =
 whitelist_externals =
     npm
 
+[testenv:py36-mysql]
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+    .[mysql]
```

Comment:
See my comment regarding a `requirements.in` file which would remove the need for this.